### PR TITLE
[MRG] Fixes to examples

### PR DESCRIPTION
**Pull request by jaeilepp**
Closes https://github.com/mne-tools/mne-python/issues/3692
Retry to see what it looks like on circle.

### CODE REVIEW

**Comment by jaeilepp** [comment]
> In plot_compute_rt_decoder.py? I get a ton of filter warnings:

I think it is because of deprecated module on sklearn0.18. We'll have to change the module eventually.

**Comment by jaeilepp** [comment]
On my machine the curve is longer than on circle https://1062-35097627-gh.circle-artifacts.com/0/home/ubuntu/mne-python/doc/_build/html/auto_examples/realtime/plot_compute_rt_decoder.html#sphx-glr-auto-examples-realtime-plot-compute-rt-decoder-py
@jasmainak is this good enough for you?

**Comment by jaeilepp** [comment]
Apparently the split view works if you provide the figures by hand https://1065-35097627-gh.circle-artifacts.com/0/home/ubuntu/mne-python/doc/_build/html/auto_tutorials/plot_point_spread.html.
Let's see if it works with just figure numbers too.

**Comment by jaeilepp** [comment]
Looks like split works too https://1066-35097627-gh.circle-artifacts.com/0/home/ubuntu/mne-python/doc/_build/html/auto_tutorials/plot_point_spread.html. Ready from my end.

**Comment by jasmainak** [comment]
@jaeilepp the example is not right. Let me look into it

**Comment by jasmainak** [comment]
@jaeilepp can you cherry pick this commit: https://github.com/jasmainak/mne-python/commit/2aff6a04770669b844740b5a6c2beb2fc3029f0f ?

Here is how the figure should look like:

expected_curve

**Comment by Eric89GXL** [comment]
> I think it is because of deprecated module on sklearn0.18.

The warnings I mentioned are from MNE-Python's filtering functions about the filter length being too long, not from `sklearn`. Looks like @jasmainak's commit should fix it, though.

**Comment by agramfort** [comment]
merge when you're happy

File: examples/realtime/plot_compute_rt_decoder.py [hunk 1]
```diff
@@ -2,7 +2,6 @@
 =======================
 Decoding real-time data
 =======================
-
```
Comment (Eric89GXL) [inline examples/realtime/plot_compute_rt_decoder.py]: this broke the CircleCI build

**Comment by jasmainak** [comment]
Thanks @jaeilepp !

**Pull request merged**